### ISSUE TITLE
Converting .ipynb to PDF Fetal format Error

### ISSUE THREAD

**Issue by coryroyce** [jupyter/nbconvert#824]
I have read various threads about PDF conversion issues but I cannot find anything that fixes my issue.

I get the below error:
> nbconvert.exporters.pdf.LatexFailed: PDF creating failed, captured latex output:
> This is XeTeX, Version 3.14159265-2.6-0.99996 (MiKTeX 2.9)
> (Fatal format file error; I'm stymied)

I'm running Jupyter on Windows 10 and have updated nbconvert & MikTex & Pandoc.

Any suggestions or additional info on trouble shooting this?
Thanks in advanced for your help on this project.

**Comment by t-makaro** [comment]
Try outputting to latex then running latex yourself. Maybe I can help with that information.

**Comment by coryroyce** [comment]
Here is a very simple Notebook that is having the error. Along with the Tex file.

[Notebook.zip](https://github.com/jupyter/nbconvert/files/2161244/Notebook.zip)

**Comment by t-makaro** [comment]
The tex file compiled perfectly fine for me, so it is valid. It looks like your issue is with your installation of MikTeX and not nbconvert.

Since you are using MikTeX, TeXworks is installed. Try opening with the tex file with that and compiling. If you've never (successfully) compiled a converted .tex file, then MikTeX will ask you to install a bunch of required packages. It is possible that the MikTeX package manger is broken as that was a known bug with one of the releases.

If, when using TeXworks, you get the UAC prompts to install some packages, accept them. The tex file should compile, though you may need to compile a second time. If that works, then retry nbconvert from either jupyter notebook or the command line and it should work.

If the MikTeX package manager is broken, or for whatever reason the tex file still won't compile. Reinstall MikTeX. [download here](https://miktex.org/download). 

Optionally, you can reinstall MikTeX using chocolatey (a package manager for windows like brew on mac or apt-get on linux). First uninstall MikTeX. Then [install chocolatey](https://chocolatey.org/install#installing-chocolatey). Then from an administrative powershell use:
```
choco install miktex
```

**Comment by coryroyce** [comment]
Based on installing the TeWorks updates I was able to get it to compile. But the nbcovert is still not working.

When opening MikTex it says there is a path issue. See attached screen shot. But since there is a "/" in the path name it can not be added to my path (Windows 10). 

Any other thoughts?

![miktex](https://user-images.githubusercontent.com/15311592/42248423-3e2b9aea-7eda-11e8-94e0-d49e178339cb.PNG)

**Comment by coryroyce** [comment]
xelatex was installed and accessible through the command prompt.

I did a complete uninstall of MikTek and reinstalled using Chocolatey as you recommended, then updated all the packages by running  the Latex file in TexWorks.

This resolved my issue and now I can save to PDF directly from Jupyter Notebooks.

I really appreciate your help, I had spent a lot of time searching and trying other solutions before this.
